conf/evolutions/016-add-schema-version.sql

```diff
@@ -0,0 +1,11 @@
+-- https://github.com/scalableminds/webknossos/pull/2904
+
+-- From now on we store schemaVersion in the database
+-- When editing the schema, please update the number both in your evolution and in schema.sql
+
+START TRANSACTION;
+CREATE TABLE webknossos.releaseInformation (
+  schemaVersion BIGINT NOT NULL
```

Comment:
Should we add a timestamp of last schema change?

Reply:
Not sure if the benefit is worth having to remember updating it in every evolution. What do you think, what would it be useful for?

Reply:
using `ADD COLUMN … TIMESTAMP DEFAULT NOW()` should solve this without extra effort

Reply:
sure but still you’d have to add `update releaseInformation set updatedTimestamp = NOW()` to every evolution.

Reply:
could be a trigger. anyways, not sure if useful at all.